src/styles/product-card.css

```diff
@@ -0,0 +1,108 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px 16px;
+
+  background-color: var(--white-accent);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__img {
+  object-fit: cover;
+  align-self: center;
+
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product-card__title {
+  margin-top: 0;
+  margin-bottom: 4px;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--main-accent);
+}
+
+.product-card__code {
+  align-self: flex-start;
+  margin-top: 0;
+  margin-bottom: 16px;
```

Comment:
you still have these extra white spaces. Please correct them.
I don`t see any other staff to correct.